kie-server-parent/kie-server-services/kie-server-services-optaplanner/src/main/java/org/kie/server/services/optaplanner/SolverServiceBase.java

```diff
@@ -515,4 +516,41 @@ private void terminateSolverEarly(SolverInstanceContext sic) {
         }
         sic.getSolver().terminateEarly();
     }
+    
+    protected boolean boostrapContinuousPlanning(String containerId, List<ContinuousPlanning> plannings) {
+        for (ContinuousPlanning planning : plannings) {
+            if (planning.accept(containerId)) {                
+                logger.info("ContinuousPlanning {} is going to be used for container {}", planning, containerId);
+                
+                String solverId = planning.getSolverId();
+                SolverInstance instance = new SolverInstance();
+                instance.setSolverConfigFile(solverId + "-solver.xml");
+                instance.setContainerId(containerId);
+                instance.setSolverId(solverId);
+                
+                logger.debug("About to start solver {}", instance);
+                createSolver(containerId, solverId, instance);
+                
+                logger.debug("Solver with id {} created successfully", solverId);
+                SolverInstanceContext solverInstance = solvers.get(solverId);
```

Comment:
This should use solvers.get(SolverInstance.getSolverInstanceKey(containerId, solverId)).